### DISCUSSION TITLE
( Spanish ) help with building

### DISCUSSION THREAD

**Discussion by eggbanana173** [nesbox/TIC-80#1587] [Q&A] [answered]
Ayuda, como hago para "buildear" TIC-80

Si ya configure todo pero cuando pongo este comando

> cmake -G "MinGW Makefiles" C:\Users\Personal\TIC-80\build
 
me sale este error 
   
> CMake Error: The source directory "C:/Users/Personal/TIC-80/build" does not appear to contain CMakeLists.txt.
    Specify --help for usage, or press the help button on the CMake GUI.

Que hago?

**Comment by nesbox** [comment]
try
```
cd C:\Users\Personal\TIC-80\build
cmake -G "MinGW Makefiles" ..
```

**Reply by eggbanana173** [reply]
pero me aparece otro

>Untracked files:
  (use "git add <file>..." to include in what will be committed)
        version.h
 nothing added to commit but untracked files present (use "git add" to track)
CMake Error: CMake was unable to find a build program corresponding to "MinGW Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMake was unable to find a build program corresponding to "MinGW Makefiles".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMAKE_C_COMPILER not set, after EnableLanguage
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
See also "C:/Users/Personal/TIC-80/build/CMakeFiles/CMakeOutput.log".

ya instale "MinGW" y nada

**Reply by eggbanana173** [reply]
si tienes la respuesta lo agradecería

**Reply by eggbanana173** [reply]
y si te lo preguntas:

>  TIC-80 0.90.1723
windows 10 Enterprise LTSC 1809 compilacion 17763.2114

**Reply by eggbanana173** [reply]
porfavor ayudaaaaaaaaa

**Reply by nesbox** [reply]
Please install Visual Studio compiler or MinGW
https://github.com/nesbox/TIC-80#windows because your CMake doesn't see any compiler installed on your system.

**Reply by eggbanana173** [reply]
pero si ya lo hice y tengo las pruebas:
<image>https://u.cubeupload.com/Star_Geoboom/Mingwlog1.png</image>
<image>https://u.cubeupload.com/Star_Geoboom/Mingwlog2.png</image>

**Reply by eggbanana173** [reply]
si, no se poner imágenes en esta pagina

**Reply by eggbanana173** [reply]
aceptaría una respuesta

**Comment by nesbox** [comment]
Pls add the `...\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin` to the system PATH variable and re-run CMake.

**Reply by eggbanana173** [reply]
pero en cual opción o comando

**Reply by eggbanana173** [reply]
perdón si lo estoy haciendo enfadar

**Reply by nesbox** [reply]
you can do it here
![image](https://user-images.githubusercontent.com/1101448/132379545-600ac3ac-1ed5-42eb-9969-b96830f72722.png)

**Reply by eggbanana173** [reply]
pero cual es la aplicación, perdón si te rompo los huevos

**Reply by eggbanana173** [reply]
lo digo porque no me aparece esa opción en"CMake"

**Reply by eggbanana173** [reply]
enserio no me aparece en "cmake"

**Comment by eggbanana173** [comment]
solo quiero una respuesta

**Reply by eggbanana173** [reply]
bueno no me quieres responder, así que mejor lo dejamos aquí la ayuda ni no respondes

**Reply by eggbanana173** [reply]
enserio necesito una respuesta

**Reply by eggbanana173** [reply]
no tranquilo ya lo vi, te informare de algún error que ocurra

**Reply by eggbanana173** [reply]
espera, tu cuales configuraciones pusiste en "cmake"

**Reply by eggbanana173** [reply]
para ver ni no configure mal en cmake

**Reply by eggbanana173** [reply]
enserio es para que es lo que hice mal (perdón por el "necroposting")

**Comment by eggbanana173** [comment, marked as answer]
no ya lo encontré gracias
